app/src/main/java/app/gamenative/ui/widget/PerformanceHudView.kt

```
@@ -0,0 +1,243 @@
+
+        addView(container)
+        handler.post(updateRunnable)
+    }
+
+    override fun onDetachedFromWindow() {
+        handler.removeCallbacks(updateRunnable)
+        super.onDetachedFromWindow()
+    }
+
+    private fun updateStats() {
+        fpsText.text = "FPS ${String.format("%.1f", fpsProvider().coerceAtLeast(0f))}"
```

detekt (1.23.8)

[warning] String.format("%.1f", fpsProvider().coerceAtLeast(0f)) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)